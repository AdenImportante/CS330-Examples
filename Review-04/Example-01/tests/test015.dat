#! /bin/sh

# Test entire program with input set 01
# disregard formatting (spaces, newlines, and letter case)

./storage items-01.txt 6 | sed 's/[ ]//g' | grep -vE "^$" | tr '[:upper:]' '[:lower:]'