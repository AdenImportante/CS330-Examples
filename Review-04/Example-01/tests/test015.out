processinglog:
(s)pickaxe
(s)shovel
(s)pickaxe
(s)speed-ii-potion
(s)tomato
(s)potatocamera
(s)potatocamera
(d)axe
(d)boots
(d)boots
playerstoragesummary:
-used100%of6slots
nme:pickaxe
dur:100
spd:1
mtl:diamond
mdr:fortune(lvl5)
nme:shovel
dur:20
spd:3
mtl:gold
mdr:unbreaking(lvl2)
nme:pickaxe
dur:100
spd:1
mtl:diamond
mdr:fortune(lvl5)
nme:speed-ii-potion
eft:spd*2
use:1
qty:1
nme:tomato
eft:hunger-10
use:2
qty:1
nme:potatocamera
eft:imagequality-97%
use:5
qty:2
